packages/plugin-devtools/src/components/TreeComponent.ts

```diff
@@ -156,7 +156,7 @@ export class TreeComponent extends OwlComponent<NodeProps> {
      */
     _getSelectionMarkersAncestors(): Set<number> {
         const selectionMarkersAncestors = new Set<number>();
-        let ancestor = this.env.editor.selection.anchor.parent;
+        let ancestor: VNode = this.env.editor.selection.anchor.parent;
```

Comment:
C'est nécessaire ça ?

Reply:
Oui sinon il dit que: "VNode' is not assignable to type 'ContainerNode'"